src/Core/OrganizationFeatures/OrganizationApiKeys/GetOrganizationApiKeyCommand.cs

```diff
@@ -17,25 +17,23 @@ public GetOrganizationApiKeyCommand(IOrganizationApiKeyRepository organizationAp
             _organizationApiKeyRepository = organizationApiKeyRepository;
         }
 
-        public async Task<OrganizationApiKey> GetOrganizationApiKeyAsync(Guid organizationId, OrganizationApiKeyType? organizationApiKeyType)
+        public async Task<OrganizationApiKey> GetOrganizationApiKeyAsync(Guid organizationId, OrganizationApiKeyType organizationApiKeyType)
         {
-            var keyType = organizationApiKeyType ?? OrganizationApiKeyType.Default;
-
-            if (!Enum.IsDefined(keyType))
+            if (!Enum.IsDefined(organizationApiKeyType))
             {
                 throw new ArgumentOutOfRangeException(nameof(organizationApiKeyType), $"Invalid value for enum {nameof(OrganizationApiKeyType)}");
             }
```

Comment:
What's the concern this is addressing?

Reply:
sigh, I really feel like this should be a built-in for the deserializer....

OK, fine here, but we need to add a tech debt to address everywhere else we have an enum